test/functional/mockedFetch.ts

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
Usually it should be <class_name>Mock, means fetchMock.ts.
But why is mocked fetch used on functional tests? Functional tests should use E2E. Probably you need this for integration tests.

Reply:
renamed